downloader/download.go

```diff
@@ -77,9 +75,7 @@ func (d *downloader) Download(files, dirs []string, readPath, writePath string,
 		fileWPath := writePath + val
 		fileRPath := readPath + val
 
-		filePath := strings.TrimPrefix(strings.TrimSuffix(fileRPath, "/"), d.rootWorkingDirectoryServer)
-
-		if filter.CheckToFilter(filePath, filterList) {
+		if filter.CheckToFilter(fileRPath, filterList) {
 			continue
```

Comment:
I might write an inline comment explaining that this continue statement skips the waitgroup add and goroutine creation. That's purely stylistic though, so I won't be offended if it isn't worth it to you.